app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connections = sqlite3.connect("cinema_db.db3")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connections.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) "
+            "VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connections.commit()
+
+    def retrieve(self) -> list:
+        actors_cursor = self._connections.execute(
+            f"SELECT id, first_name, last_name FROM {self.table_name}"
```

Comment:
It's better to always start clauses such as `FROM`, `WHERE`, etc from a new line when writing raw SQL.